test/e2e/util.go

```diff
@@ -631,12 +642,20 @@ func Diff(oldPods *api.PodList, curPods *api.PodList) PodDiff {
 // It will waits for all pods it spawns to become "Running".
```

Comment:
The line above this starts with "RunRC Launches ...".  I'm fine changing It->RunRC.  The comment for this function could stand to be cleaned up a bit more.  There's some grammar issues in there.  Happy to do that in my next push.